terraform/main.tf

```diff
@@ -113,10 +114,35 @@ module "pool_agent4-ws2019-vs2019" {
   AZURERM_VIRTUAL_NETWORK_MAIN_NAME      = "${data.azurerm_virtual_network.data.name}"
   AZURERM_SUBNET_ID                      = "${data.azurerm_subnet.data.id}"
   AZURERM_NETWORK_SECURITY_GROUP_MAIN_ID = "${data.azurerm_network_security_group.data.id}"
-  VM                                     = "${element(var.SERVERNAMES, 3)}"
+  VM                                     = "${element(var.SERVERNAMES, 3)}"  
   BRANCH                                 = "${var.BRANCH}"
   TAGS                                   = "${var.TAGS}"
   vm_name                                = "MAZDO${upper(var.ENVIRONMENT)}${element(var.SERVERNAMES, 3)}"
   vm_size                                = "${var.VM_SIZE}"
   run_date                               = "${var.RUN_DATE}"
 }
+
+
+
+module "pool_agent5-ws2019-vs2019" {
```

Comment:
I am not 100% sure this is the best option, but I am not sure what the alternative might be.

we dont need 2 ws2019 boxes so the pool_agent5 module call can be removed.

else you will need another public ip.

Reply:
OK. I will remove pool_Agent5. Are the public IPs not configured through TF? That being said, I don't think the windows agents need a public IP.

Reply:
this isnt done in terraform, it was never an intended component, all that is required is to disassociate and associate the ip addresses as the new boxes come up.

I have some scripts I should really share :)